lake-manifest.json

```diff
@@ -72,42 +72,6 @@
    "manifestFile": "lake-manifest.json",
    "inputRev": null,
    "inherited": false,
-   "configFile": "lakefile.lean"},
```

Comment:
I'm a little confused as to why your Lean instances decided to delete this stuff from the manifest. It is probably best to revert this change.